javascript-light-bulb/main.js

```diff
@@ -0,0 +1,18 @@
+var statusOfLight = true;
```

Comment:
A better name for this variable would be `isLightOn`. That makes the conditional statement below make more sense.